astropy/io/fits/tests/test_connect.py

```diff
@@ -835,35 +769,10 @@ def test_fits_mixins_as_one(table_cls, tmpdir):
     """Test write/read all cols at once and validate intermediate column names"""
     filename = str(tmpdir.join('test_simple.fits'))
     names = sorted(mixin_cols)
-
-    serialized_names = ['ang',
-                        'cr.x', 'cr.y', 'cr.z',
-                        'dt.jd1', 'dt.jd2',
-                        'el2.x', 'el2.y', 'el2.z',
-                        'lat',
-                        'lon',
-                        'sc.ra', 'sc.dec',
-                        'scd.ra', 'scd.dec', 'scd.distance',
-                        'scd.obstime.jd1', 'scd.obstime.jd2',
-                        'scdc.x', 'scdc.y', 'scdc.z',
-                        'scdc.obstime.jd1', 'scdc.obstime.jd2',
-                        'scpm.ra', 'scpm.dec', 'scpm.distance',
-                        'scpm.pm_ra_cosdec', 'scpm.pm_dec',
-                        'scpmrv.ra', 'scpmrv.dec', 'scpmrv.distance',
-                        'scpmrv.pm_ra_cosdec', 'scpmrv.pm_dec',
-                        'scpmrv.radial_velocity',
-                        'scrv.ra', 'scrv.dec', 'scrv.distance',
-                        'scrv.radial_velocity',
-                        'sd.d_lon_coslat', 'sd.d_lat', 'sd.d_distance',
-                        'sr.lon', 'sr.lat', 'sr.distance',
-                        'srd.lon', 'srd.lat', 'srd.distance',
-                        'srd.differentials.s.d_lon_coslat',
-                        'srd.differentials.s.d_lat',
-                        'srd.differentials.s.d_distance',
-                        'tm',  # serialize_method is formatted_value
-                        'x',
-                        ]
-
+    # FITS stores times directly, so we just get the column back.
+    all_serialized_names = sum((([name] if isinstance(mixin_cols[name], Time)
```

Comment:
Agreed that it is a bit of a trick - since it really is not worth getting whoever looks at it next to think what the heck is happening here, I just replaced it with an explicit loop.